xplenty/xplenty_api.py

```diff
@@ -305,94 +304,99 @@ def delete(self, url):
         return json.loads(resp.read())
 
     def _join_url(self, method):
-        _url = API_URL % ( self.account_id )
-        url = urljoin(_url , method )
+        _url = API_URL % (self.account_id)
+        url = urljoin(_url, method)
         return url
 
+    def _add_authorization_header(self, request, api_key: str):
+        base64string = base64.b64encode(bytes('%s' % api_key, 'ascii'))
+        request.add_header("Authorization", "Basic %s" % base64string.decode('utf-8'))
+
```

Comment:
Reading the documentation it seems they would expect base 64 encoded, and have it be the api key followed by a colon.

https://github.com/xplenty/xplenty-api-doc-v2#authentication

But does this work as is? If you functionally tested and this works then cool

Reply:
I did finally test this locally 🤭 , it was working without the colon and thats how the original xplenty-python-api was written but I added it anyhow (works too).